src/Bicep.LangServer.IntegrationTests/HoverTests.cs

```diff
@@ -62,9 +62,12 @@ public async Task HoveringOverSymbolReferencesAndDeclarationsShouldProduceHovers
 
             foreach (SyntaxBase symbolReference in symbolReferences)
             {
-                var syntaxPosition = symbolReference is ITopLevelDeclarationSyntax declaration
-                    ? declaration.Keyword.Span.Position
-                    : symbolReference.Span.Position;
+                var syntaxPosition = symbolReference switch
+                {
+                    ITopLevelDeclarationSyntax d => d.Keyword.Span.Position,
+                    ResourceAccessSyntax r => r.ResourceName.Span.Position,
+                    _ => symbolReference.Span.Position,
+                };
```

Comment:
I tried to update this to use `IntegrationTestHelper.GetPosition` but then it started breaking for other cases unrelated to my changes.